tools/sdk-cli/src/commands.rs

```diff
@@ -144,6 +144,16 @@ pub(crate) enum Commands {
         /// Optional offset in payments
         #[clap(short = 'o', long = "offset")]
         offset: Option<u32>,
+
+        /// Optional metadata filter, in the form of json_path:json_value,...
+        #[clap(short = 'm', long = "metadata", num_args = 1.., value_delimiter = ',')]
```

Comment:
Can't the value_delimiter `,` and the split token `:`  be a valid components of json_value? 

E.g. `supportsNested:{"value":[1,2]}` or `supportsNested.value:[1,2]`

Reply:
True, that'd be the case for objects/arrays. Perhaps a delimiter isn't necessary, and we can use `-m` repeatedly. Concerting the `:`, I'll just join the slices into a single string. That should avoid the issue.